routes/profile/getUserProfile.js

```diff
@@ -0,0 +1,37 @@
+/*
+Purpose of API: This API will retrieve the user's existing profile, if there is one and display it on their myProfile page.  
+*/
+const express = require('express');
+const router = express.Router();
+const mysql = require('mysql');
+const config = require('../../config.js');
+
+router.post('/api/getUserProfile', (req, res) => {
+
+	let connection = mysql.createConnection(config);
+	let sql = "SELECT * FROM shchowdh.userProfiles WHERE userID = ? AND submissionDate IN (SELECT max(submissionDate) FROM shchowdh.userProfiles)"; 
```

Comment:
ideally, each user has one row in the new table (1:1 relation). So the query should be:
`SELECT * FROM shchowdh.userProfiles WHERE userID = ?`